package.json

```diff
@@ -37,7 +37,7 @@
     "localforage-observable": "2.0.0",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#5a06d1af25ca48927b9efe0d9c42c01c84e7839b",
-    "mattermost-redux": "github:mattermost/mattermost-redux#3c07f225aae53e91d55de9ccd23d4eb4cabc98f0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#64e27f1dd71171dacc53a08fc910acc2d03dbdc9",
```

Comment:
Is there a corresponding redux PR for this, or are you just updating the ref to a more recent version of the redux branch?

Reply:
This is just a branch update:
https://github.com/mattermost/mattermost-redux/commit/64e27f1dd71171dacc53a08fc910acc2d03dbdc9